src/logicblocks/event/processing/broker/difference.py

```diff
@@ -14,7 +14,7 @@
 class EventSubscriptionChange:
     group: str
     id: str
-    event_source: EventSourceIdentifier
+    event_source: EventSequenceIdentifier
```

Comment:
Why did you change this one?

Reply:
Maybe we need to chat this through properly - but change it back and see the type errors and you can see why. I wasn't sure the best direction to resolve those errors, so i went with this to get something stable up to show you